### DISCUSSION TITLE
7 Days to Die new Web Dashboard integration.

### DISCUSSION THREAD

**Discussion by ad3d0** [gameserverapp/Platform#464] [Suggestions]
**With a21 TFP started providing built in API and Web Dashboard which uses that API to let server administrators easy access to whole range of server and player statistics.** 
There are 4 new server config parameters related to that web dashboard:
`<property name="WebDashboardEnabled" value="true"/>`	
`<property name="WebDashboardPort" value="8080"/>`
`<property name="WebDashboardUrl" value=""/>`
`<property name="EnableMapRendering" value="true"/>`
**So when a21 is released to main stable build it would be great to have those 4 added to default config instead of outdated:**
`<property name="ControlPanelEnabled" value="false"/>`			
`<property name="ControlPanelPort" value="8080"/>	`		
`<property name="ControlPanelPassword" value="CHANGEME"/>`
**And the WebDashboardPort value could be added to the server configuration among other ports, so users would not have to manually open those.**

**Comment by ad3d0** [comment]
a21 was pushed to stable branch today.

**Reply by mvaessen** [reply]
Hi, thanks for the info. Do you have any api docs about how GSA would be able to get the information from their API?

**Comment by mvaessen** [comment]
Thanks again for posting! I'll make sure to update the default configs and add the extra port
